env/env.cc

```diff
@@ -22,6 +23,20 @@ namespace rocksdb {
 Env::~Env() {
 }
 
+Status Env::NewLogger(const std::string& fname,
+                      std::shared_ptr<Logger>* result) {
```

Comment:
Can we make it a separate helper function, which Env::NewLogger() calls too?

Reply:
Done. Created a new public helper method NewEnvLogger().